contributors/devel/api-conventions.md

```diff
@@ -272,7 +272,7 @@ desired state.
 
 The PUT and POST verbs on objects MUST ignore the "status" values, to avoid
 accidentally overwriting the status in read-modify-write scenarios. A `/status`
-subresource MUST be provided to enable system components to update statuses of
+subresource MUST be provided to enable system components to update statuses and metadata of
```

Comment:
> If I were to create a new /status subresource (this is with respect to CustomResources), do you think it should touch metadata then?
> 
> cc @deads2k

No.  I don't think `/status` should touch objectmeta.  See https://github.com/kubernetes/kubernetes/pull/45552 as a possible means of preventing it.